student-resumes/skimpysquidman-resume/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html> 
+<html> 
+    <style>
+        h1 {
+            font-weight:bolder;
+            font-size: 50px;
+            font-family: 'Times New Roman', Times, serif;
+        }
+        h2 {
+            font-weight:bold;
+            font-size: 30px;
+            font-family: 'Times New Roman', Times, serif;
+        }
+
+        h3 {
+            font-size: 20px;
+            font-family:Arial, Helvetica, sans-serif;
+        }
+
+        .bullets {
+            font-size: 16px;
+            font-family: Arial, Helvetica, sans-serif;
+        }
+    </style>
+
+<head> 
+    <title>Jeremy Honeyman's HTML Resume
+    </title> 
+</head> 
+
+<body style="margin: 10%;">
+    <h1 style="text-align:center"><u>Jeremy Honeyman</u></h1> 
+    <p style="text-align:center; font-family: Arial, Helvetica, sans-serif; font-size: 20px;">jeremyhoneyman@outlook.com &#128148; 604-329-1653 &#128148; Vancouver, BC</p>
+<hr style="height: 3px; background-color: black;">
+    <h2><u>Work Experience</u></h2>
+    <h3>Hootsuite<span style="float:right">July 2023 - Present</span><br><em style="font-weight: normal;">Marketing Automation Specialist<span style="float:right">Vancouver, BC</span></em></h3>
+    <ul class="bullets">
+        <li>Collaborate with email teams to route leads from email campaigns into self maintained nurture flows present in our Marketo instance</li>
+        <li>Integrate new marketing technologies into Hootsuite's automation system; reducing costs and simplifying processeses while maintaining a consistent lead stream for sales</li>
+        <li>Design and implement technical solutions in Hootsuite's Marketo and Salesforce instances</li>
+        <li>As an appointed member of Hootsuite's privacy team: maintain & ensure the privacy of leads in a database of over 9,000,000 people according to CASL and GDPR standards</li>
```

Comment:
Just testing how comments work, I think you should be able to see this right? This is where I try to justify my iffy code to my boss at my future job? LMAO jk.

Reply:
I SEE EVERYTHING